app/src/main/java/com/duckduckgo/app/browser/omnibar/animations/BrowserTrackersAnimatorHelper.kt

```diff
@@ -43,7 +44,22 @@ interface BrowserTrackersAnimatorHelper {
         entities: List<Entity>?
     )
 
-    fun createCookiesAnimation(context: Context)
+    /**
+     * This method creates or enqueues cookie consent animation.
+     *
+     * @param context indicates if animation should pause, at 50% of progress, until {@link finishPartialTrackerAnimation()} is called.
```

Comment:
Looks like this parameter has the wrong description

Reply:
Actually we don't need context here, we have it as constructor param.